Dockerfile

```
@@ -0,0 +1,11 @@
+FROM node:18-slim
+
+WORKDIR /usr/src/app
+COPY package*.json .
+RUN npm install
+COPY . .
+RUN npm run build
+RUN npm install -g .
+RUN chmod +x entrypoint.sh
+
+ENTRYPOINT [ "./entrypoint.sh" ]
```

I think there should be a blank line at the end of the file

```suggestion
ENTRYPOINT [ "./entrypoint.sh" ]

```